doc/classes/XRInterface.xml

```diff
@@ -198,6 +198,15 @@
 		<member name="xr_play_area_mode" type="int" setter="set_play_area_mode" getter="get_play_area_mode" enum="XRInterface.PlayAreaMode" default="0">
 			The play area mode for this interface.
 		</member>
+		<member name="xr_vrs_min_radius" type="float" setter="set_vrs_min_radius" getter="get_vrs_min_radius" default="20.0">
+			The minimum radius around the focal point where full quality is guaranteed if VRS is used.
```

Comment:
This one I have a good feeling is in degrees?
```suggestion
			The minimum radius (in degrees) around the focal point where full quality is guaranteed when VRS is enabled.
```

Reply:
Radius is usually a distance, not an angle, so I'd personally suspect it's not degrees or radians.

Reply:
I see, it's distance in units again. Okiedokie!

Reply:
Yup, distance, in pixels to be exact.